### ISSUE TITLE
As a manager, I want to generates unassigned shifts to the schedule so that my selected employees can view it 

### ISSUE THREAD

**Issue by LinhNguyenLe2109** [leoschwartz/ShiftManagementTool#46]
# Description
As a manager, I should have the option to create shifts that aren't assigned to any employee. 

# Acceptance Criteria
- Manager can access the page for creating unedited shifts
- Unassigned shift can be created & saved.

# Testing
- Create new unassigned shift
- Verify that shift exists on database

**Comment by pstaleystudent** [comment]
Dependent on #51.
Actually viewing unassigned shifts currently not in scope? Will discuss today.
